README.md

```diff
@@ -1,5 +1,5 @@
 # apache2 Cookbook
-[![CircleCI](https://circleci.com/gh/sous-chefs/apache2.svg?style=svg)](https://circleci.com/gh/sous-chefs/apache2) [![Cookbook Version](https://img.shields.io/cookbook/v/apache2.svg?style=flat)](https://supermarket.chef.io/cookbooks/apache2) [![License](https://img.shields.io/badge/license-apache2_2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
no more badge?

Reply:
possibly a weird merge, I'll fix that in the next commit.